test/releasers/php.ts

```diff
@@ -0,0 +1,139 @@
+      packageName: 'yoshi-php',
+    });
+
+    sandbox
+      .stub(releasePR.gh, 'getDefaultBranch')
+      .returns(Promise.resolve('master'));
+
+    // No open release PRs, so create a new release PR
+    sandbox
+      .stub(releasePR.gh, 'findOpenReleasePRs')
+      .returns(Promise.resolve([]));
+
+    // Indicates that there are no PRs currently waiting to be released:
+    sandbox
+      .stub(releasePR.gh, 'findMergedReleasePR')
+      .returns(Promise.resolve(undefined));
+
+    sandbox.stub(releasePR, 'latestTag').returns(
+      Promise.resolve({
+        name: 'v0.20.3',
+        sha: 'bf69d0f204474b88b3f8b5a72a392129d16a3929',
+        version: '0.20.3',
+      })
+    );
+
+    const graphql = JSON.parse(
+      readFileSync(
+        resolve(fixturesPath, 'commits-yoshi-php-monorepo.json'),
+        'utf8'
+      )
+    );
+    const req = nock('https://api.github.com')
+      // now we fetch the commits via the graphql API;
+      // note they will be truncated to just before the tag's sha.
+      .post('/graphql', () => {
+        return true;
+      })
+      .reply(200, {
+        data: graphql,
+      });
+
+    const getFileContentsStub = sandbox.stub(
+      releasePR.gh,
+      'getFileContentsOnBranch'
+    );
+    getFileContentsStub
+      .withArgs('AutoMl/composer.json', 'master')
+      .resolves(buildGitHubFileRaw('{"name": "automl"}'));
+    getFileContentsStub
+      .withArgs('AutoMl/VERSION', 'master')
+      .resolves(buildGitHubFileRaw('1.8.3'));
+    getFileContentsStub
+      .withArgs('Datastore/composer.json', 'master')
+      .resolves(buildGitHubFileRaw('{"name": "datastore"}'));
+    getFileContentsStub
+      .withArgs('Datastore/VERSION', 'master')
+      .resolves(buildGitHubFileRaw('2.0.0'));
+    getFileContentsStub
+      .withArgs('PubSub/composer.json', 'master')
+      .resolves(buildGitHubFileRaw('{"name": "pubsub"}'));
+    getFileContentsStub
+      .withArgs('PubSub/VERSION', 'master')
+      .resolves(buildGitHubFileRaw('1.0.1'));
+    getFileContentsStub
+      .withArgs('Speech/composer.json', 'master')
+      .resolves(buildGitHubFileRaw('{"name": "speech"}'));
+    getFileContentsStub
+      .withArgs('Speech/VERSION', 'master')
+      .resolves(buildGitHubFileRaw('1.0.0'));
+    getFileContentsStub
+      .withArgs('WebSecurityScanner/composer.json', 'master')
+      .resolves(buildGitHubFileRaw('{"name": "websecurityscanner"}'));
+    getFileContentsStub
+      .withArgs('WebSecurityScanner/VERSION', 'master')
+      .resolves(buildGitHubFileRaw('0.8.0'));
+    getFileContentsStub
+      .withArgs('composer.json', 'master')
+      .resolves(buildGitHubFileRaw('{"replace": {}}'));
+    getFileContentsStub
+      .withArgs('docs/manifest.json', 'master')
+      .resolves(
+        buildGitHubFileRaw(
+          '{"modules": [{"name": "google/cloud", "versions": []}, {"name": "datastore", "versions": []}]}'
+        )
+      );
+    getFileContentsStub.rejects(
```

Comment:
I'm not sure what this 404 correlates too, perhaps trimming down the unit test to the simple PHP case we can drop this.

Reply:
removed